src/ConfigProvider.php

```diff
@@ -26,7 +27,13 @@ class ConfigProvider
 {
     public function __invoke(): array
     {
-        $config = PHP_SAPI === 'cli' && extension_loaded('swoole')
+        if (! extension_loaded('swoole') && ! extension_loaded('openswoole')) {
+            throw new ExtensionNotLoadedException(
+                'One of either the swoole or openswoole extensions must be loaded to use mezzio/mezzio-swoole'
```

Comment:
I think links to the related websites / documentation would be good here; for both extensions.

Reply:
Done.